airbyte-integrations/connectors/source-onesignal/metadata.yaml

```diff
@@ -1,24 +1,26 @@
 data:
-  ab_internal:
-    ql: 200
-    sl: 100
+  allowedHosts:
+    hosts:
+      - "onesignal.com"
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
   connectorSubtype: api
   connectorType: source
   definitionId: bb6afd81-87d5-47e3-97c4-e2c2901b1cf8
-  dockerImageTag: 1.0.1
+  dockerImageTag: 1.1.0
   dockerRepository: airbyte/source-onesignal
   documentationUrl: https://docs.airbyte.com/integrations/sources/onesignal
   githubIssueLabel: source-onesignal
   icon: onesignal.svg
   license: MIT
   name: OneSignal
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: 2023-08-12
   releaseStage: alpha
   supportLevel: community
   tags:
-    - language:python
+    - language:lowcode
+  supportLevel: certified
```

Comment:
```suggestion
```

Reply:
Oops, meant to make this change on a different PR sorry! reverted in https://github.com/airbytehq/airbyte/pull/28941/commits/275446f72595f9413969c3d8af368748a087910f